### ISSUE TITLE
chart{events: { load: function() {alert('hello');}}.... Not Firing

### ISSUE THREAD

**Issue by queejie** [pablojim/highcharts-ng#203]
Thank you for this effort!  I'm a long-time developer, but new to AngularJS.

I saw another issue with a similar title, but it had to do with the func attribute.  The alert in the load function is never called.

I've read the part in the README about the highchartsNgConfig several times, and I've looked through the example source code and other fiddles.  I just can't understand the distinction being made in the document, and I can't find an example that uses that construct.  I'm not sure if I fall into the category of people who are making that mistake since I can't understand what the mistake is.  

I am creating a config that is referenced in the tag:

<highchart id="productBreak_{{product.taskProductId}}" config="product.breakDownConfig" class="span12"></highchart>

That config contains the chart{events: { load: function() {alert('hello');}}... declaration.

**Comment by Brightergy-AuturBurianek** [comment]
@pablojim I have met same problem.
============================================================
I have discussed with Highcharts support team for my programming.
Here is the jsfiddle link that Highcharts support team provided me.
http://jsfiddle.net/t7rzjax6/6/
Let me explain about above jsfiddle demo.
When I click 'Add point' button, on the left chart will have new point and yAxis max value will be changed. At that time, the chart.load event is fired and they get left chart yAxis max value and set it into right chart yAxis max value.
It works well on jQuery version.
============================================================

I simulate it on angular version, the chart.load event is not fired when add the new points (the chart.load event is only once fired, when chart initialize).
Here is the jsfiddle demo link for angular version http://jsfiddle.net/qtehx8en/2/
Could you help me as soon as possible ?

P.S. Please check care of http://jsfiddle.net/t7rzjax6/6/ (My purpose is just it).
Looking forward to hear from you.
Thanks.
